### ISSUE TITLE
Save Taxonomy Pod After Renaming Field Label/Name Produces SQL Error

### ISSUE THREAD

**Issue by qrahaman** [pods-framework/pods#4613]
I have two Pod-created taxonomies: Difficulty and Property.  In the Property taxonomy, I have defined 5 fields.  I tried to rename both the label and name of one of those fields from "thickness" to "width" and the following error shown in the screenshot was produce upon attempting to save the pod.  

![screenshot 2017-12-06 04 42 27](https://user-images.githubusercontent.com/1299653/33662728-cb07c88c-da41-11e7-906a-1675e5c7614f.png)

**Comment by jimtrue** [comment]
Thanks for catching this one @qrahaman !

I was able to instantly verify. Steps to reproduce:

1) Add a field to a taxonomy, save it, and then Save the Pod.
2) Change the name of the field, save it. Attempt to Save the Pod.
3) Immediate exact same error.

**Comment by sc0ttkclark** [comment]
We need to be joining the wp_term_taxonomy table instead here

**Comment by qrahaman** [comment]
I just tried renaming/saving the field (“thickness” to “width”) in my Property taxonomy Pod.  No SQL error.  So, it worked there.  

However, in the other pods that I have associated this taxonomy, when I click on the Properties taxonomy in the menu for one of those pods, I see the list of categories for Property (all five of them).  Unfortunately, “thickness” has not been changed to “width”.

![screenshot 2017-12-06 09 13 17](https://user-images.githubusercontent.com/1299653/33674918-d51647c6-da65-11e7-8b7c-392d7930503c.png)

![screenshot 2017-12-06 09 13 45](https://user-images.githubusercontent.com/1299653/33674923-d9dd7b58-da65-11e7-942c-dd9fb76eb59f.png)
